legal-api/src/legal_api/models/alternate_name.py

```diff
@@ -182,6 +182,42 @@ def is_owned_by_colin_entity(self):
         """Return if owned by colin entity."""
         return bool(self.colin_entity)
 
+    @property
+    def owner_data_json(self):
+        """Return if owner data."""
+        json = {
+            "officer": {},
+            "roles": [{"appointmentDate": "2000-01-03", "cessationDate": None, "roleType": "Proprietor"}],
```

Comment:
Populate `appointmentDate` with `self.startDate`

Reply:
fixed

Reply:
Can you give me a screenshot what appointmentDate looks like via postman call?